Source/RxCocoa/reachedBottom.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  reachedBottom.swift
+//  RxSwiftExt-iOS
+//
+//  Created by Joan Disho on 17.08.18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+extension Reactive where Base: UIScrollView {
+    /**
+     Shows if the the bottom of the UIScrollView is reached.
+
+     - parameter offset: A threshhold indicating the bottom.
+     - returns: A boolean value showing if the bottom is reached or not.
+     */
+    
+    func reachedBottom(_ offset: CGFloat = 0.0) -> ControlEvent<Bool> {
```

Comment:
I'm not sure if we should use a ControlEvent or a Signal in this case. It's not really a control event (e.g. one that stems from a UIControl.Event).

Also, IRT the implementation - it mostly looks O.K but missing all the tests, playground page, etc.

Reply:
Hey @freak4pc `Signal` looks fine for you? I am ready to provide tests & playground page if current implementation is ok

Reply:
I think `.reachedBottom` is the most popular boilerplate thing which have to be part of this lib

Reply:
@MortyMerr I think observable would me more suitable. Please feel free to change it and add some tests. 😊

Reply:
@jdisho why not Signal?
Working on it right now, ready to discuss

Reply:
I think it should definitely be a `Signal` or `ControlEvent` as they both represent UI events. These traits also provide a main-thread guarantee (which you'd want for a UI Element).

We could use `Observable` but that's more generic and not "specific". I think `Signal` makes most sense.

Reply:
Hey @MortyMerr :) 

OSS is an open discussion, we're not commercially developing this project :)

I'm not a Dictator and don't possess ultimate knowledge; I experiment, and sometimes change my mind and I think that's OK ;-) Some more patience would be appreciated. 

My initial instinct and current is to use Signal because of the guarantees.

Reply:
Hi everyone just to give my opinion here I have one question: 

- When you tap a button what is it? = `ControlEvent`
- When you select a cell what is it? = `ControlEvent`
- When `didScroll` what is it? = `ControlEvent`

So this should be a `ControlEvent` IMHO because you a scrolling and you reached the `Bottom`of the page due to your scrolling actions which was a `ControlEvent`. However `Signal` or `Driver` is also fine, but `ControlEvent` fits better.

Reply:
@bobgodwinx I have nothing to argue against this. I also prefer `ControlEvent`. @freak4pc what do u think?

Reply:
OK, after looking into Core source code I think we can go with `ControlEvent` as well. 

FYI though, note that most of these (if not all) were added before Signal was introduced. So they don't all necessarily fit a ControlEvent. Anyways, sure, let's go with it.